vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.h

```diff
@@ -16,6 +16,19 @@ struct VirtualTextShadowNode final : public ShadowNodeBase {
   TransformableText transformableText{};
 
   void AddView(ShadowNode &child, int64_t index) override;
+
+  bool m_isHighlighted = false;
+  folly::dynamic m_ColorValue = nullptr;
+  size_t m_length = 0;
+
+  typedef struct HighlightData {
+    std::vector<std::unique_ptr<HighlightData>> data;
+    size_t spanIdx = 0;
+    bool isHighlighted = false;
+    folly::dynamic color;
+  } HighlightData;
+
+  std::unique_ptr<HighlightData> m_highlightData = std::make_unique<HighlightData>();
 };
```

Comment:
don't allocate if you don't need to

Reply:
So you made me think a little here: I think I might be lazy-initialize it only when a Span has child Spans - ie. at least 2 levels of nesting - because that's when I really need to use it, but would have to change how I pass around the color to check for inheritance.
I'll try this :)